corehq/apps/app_manager/suite_xml/features/case_tiles.py

```diff
@@ -47,14 +39,18 @@ def filepath(self):
 
 @memoized
 def case_tile_template_config(template):
+    if template == CUSTOM:
+        return CaseTileTemplateConfig(**{})
```

Comment:
Nit: this should be the same as
```suggestion
        return CaseTileTemplateConfig()
```